### PR TITLE
blog: SSO and OAuth

### PR DESCRIPTION
**Pull request by mvollmer**
Rendered: https://mvollmer.github.io/cockpit-project.github.io/blog/sso-oauth.html

### CODE REVIEW

File: _posts/2018-05-28-sso-oauth.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: OAuth! Single Sign On! Seamless embedding!
```
Comment (martinpitt) [inline _posts/2018-05-28-sso-oauth.md]: Seems a bit over the top, but okay :)

File: _posts/2018-05-28-sso-oauth.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+But, you have just logged into your management pane which holds root
+power over all your machines, and there shouldn't be any need to type
+in your credentials for the individual machine again.
+
+With ManageIQ, this is the case: It somehow manages to open Cockpit
+seamlessly without any additional username and passwords.  Foreman
+can't do this and puts on you on the login screen of Cockpit.
+
+What's the difference and can't we improve Foreman?
```
Comment (martinpitt) [inline _posts/2018-05-28-sso-oauth.md]: "can" reads better here IMHO.

Comment (mvollmer) [reply _posts/2018-05-28-sso-oauth.md]: Agreed.

**Comment by mvollmer** [comment]
@martinpitt ptal kthx

**Pull request merged**